config/routes.rb

```diff
@@ -1,4 +1,8 @@
 Rails.application.routes.draw do
+  devise_for :users
+  resources :users, :only => [:index, :show]
   resources :books
+  mount LetterOpenerWeb::Engine, at: "/letter_opener" if Rails.env.development?
+  root to: 'books#index'
```

Comment:
好みの問題なので修正は必須ではないですが、例えば、

```suggestion
  devise_for :users

  root to: 'books#index'

  resources :users, :only => [:index, :show]
  resources :books

  mount LetterOpenerWeb::Engine, at: "/letter_opener" if Rails.env.development?
```

のようになっていると見やすいかなと思いました。

Reply:
こちらご提示いただいたように修正しました。
見やすいです・・・